nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc

```diff
@@ -12,6 +12,36 @@ These release notes track the development of the {descheduler-operator}.
 
 For more information, see xref:../../../nodes/scheduling/descheduler/index.adoc#nodes-descheduler-about_nodes-descheduler-about[About the descheduler].
 
+[id="descheduler-operator-release-notes-5.0.1"]
+== Release notes for {descheduler-operator} 5.0.1
+
+Issued: 2024-07-01
+
+The following advisory is available for the {descheduler-operator} 5.0.1:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3617[RHSA-2024:3617]
+
+[id="descheduler-operator-5.0.1-notable-changes"]
+=== New features and enhancements
+
+* You can now install and use the {descheduler-operator} in an {product-title} cluster running in FIPS mode.
++
+[IMPORTANT]
+====
+To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
```

Comment:
@ingvagabund I pulled this from where we mentioned FIPS compliance in another operator's release notes: https://docs.openshift.com/container-platform/4.15/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-1-3-1-new-features-and-enhancements

Let me know if this looks correct to include here as well.

Reply:
Just FYI that I ended up replacing this with a new snippet that is used across the docs elsewhere. It introduces an additional sentence: 

> When running Red Hat Enterprise Linux (RHEL) or Red Hat Enterprise Linux CoreOS (RHCOS) booted in FIPS mode, OpenShift Container Platform core components use the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.